google_drive/google_calendar.py

```diff
@@ -33,7 +33,7 @@ def _(text):
 
 # Classes ###########################################################
 @XBlock.needs("i18n")  # pylint: disable=too-many-ancestors
-class GoogleCalendarBlock(XBlock, PublishEventMixin):
+class GoogleCalendarBlock(XBlock, PublishEventMixin):  # pylint: disable=too-many-ancestors
```

Comment:
you can disable this warning altogether by adding it in `pylintrc` file's disable section.